doc/source/whatsnew/v0.23.0.txt

```diff
@@ -119,6 +119,47 @@ Current Behavior
 
     s.rank(na_option='top')
 
+
+.. _whatsnew_0230.enhancements.assign_dependent:
+
+``.assign()`` accepts dependent arguments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`DataFrame.assign()` now accepts dependent kwargs for python version later than 3.6 (see also `PEP 468
+<https://www.python.org/dev/peps/pep-0468/>`_). Now the keyword-value pairs passed to `.assign()` may depend on their predecessors if the values are callables. (:issue:`14207`)
```

Comment:
"keyword-value pairs" -> "keyword arguments".

I'm not sure if "predecessors" is the right word. How about

"Later keyword arguments may now refer to earlier ones. (:issue:`14207`)"

Reply:
Sounds indem bester. Should i Dropbox the reference to callables?

Reply:
Maybe keep "callables", so

    "Later keyword arguments may now refer to earlier ones if the argument is a callable. (:issue:`14207`)"